doozer/doozerlib/cli/scan_osh.py

```diff
@@ -623,12 +656,8 @@ async def run(self):
         if self.create_jira_tickets:
             brew_components = {}
             for nvr in all_nvrs:
-                if "container" not in nvr:
-                    # Exclude all non-container images
-                    continue
-
-                if not nvr.endswith(".stream"):
-                    # Exclude all non stream builds
+                if nvr.endswith(".test"):
```

Comment:
I think we have an existing method to extract out assembly name from an nvr, we might be better off relying on that

Reply:
Also since rpms end with `.elX`

Reply:
Well I'm only skipping test builds. Doesn't rpm test build also end with `.test`?

Reply:
they would end with `.assembly.elX`